src/main/java/org/takes/rs/RsWithType.java

```diff
@@ -47,10 +52,12 @@
     public RsWithType(final Response res, final CharSequence type) {
         super(
             new RsWithHeader(
-                new RsWithStatus(res, HttpURLConnection.HTTP_OK),
-                "Content-Type", type
+                new RsWithoutHeader(
+                    new RsWithStatus(res, HttpURLConnection.HTTP_OK),
+                    HEADER
+            ),
```

Comment:
@Bertram25 Hmmm. Strange error. Have you tried inlining the params in line 59 with this one and indenting the paren?